arches_querysets/rest_framework/serializers.py

```diff
@@ -46,6 +47,8 @@ def _make_tile_serializer(
         "permitted_nodes": permitted_nodes,
         "graph_slug": slug,
         "root_node": nodegroup_alias,
+        # TODO(arches_version): remove nodegroup_alias_lookup
```

Comment:
After talking with Cyrus - we'll target Arches 9.x for removing this 7.x code support - same for other versions of this comment.